components/UserSavedAddress.vue

```diff
@@ -46,12 +46,12 @@
           <template #label>
             <div class="radio-button">
               <div v-if="isDefaultAddress()" class="is-primary">{{ $t("Primary") }}</div>
-              <p>{{ firstName }} {{ lastNameOrSurname }}</p>
+              <p>{{ firstName }} {{ lastName }}</p>
               <p>{{ address1 }} {{ address2 }}</p>
-              <p>{{ postalOrZipCode }}</p>
+              <p>{{ zipCode }}</p>
               <p>
-                {{ cityOrTown }},
-                {{ stateOrProvince }}
+                {{ city }},
+                {{ state }}
```

Comment:
It should be lastNameOrSurname  instead of lastName 
It should be postalOrZipCode   instead of zipCode
It should be cityOrTown instead of city
It should be stateOrProvinceinstead of state 

Why are we reverting it back? havn't we rebased code?

Reply:
Done